apps/cowswap-frontend/src/modules/ordersTable/pure/ReceiptModal/OrderTypeField.tsx

```diff
@@ -4,13 +4,16 @@ import * as styledEl from './styled'
 
 export type Props = {
   order: ParsedOrder
+  isTwapOrder: boolean
 }
 
-export function OrderTypeField({ order }: Props) {
+export function OrderTypeField({ order, isTwapOrder }: Props) {
+  const orderClass = isTwapOrder ? 'TWAP' : order.class
+
   return (
     <styledEl.Value>
       <styledEl.OrderTypeValue>
-        {order.class} {order.kind} order {order.partiallyFillable ? '(Partially fillable)' : '(Fill or Kill)'}
+        {orderClass} {order.kind} order {order.partiallyFillable ? '(Partially fillable)' : '(Fill or Kill)'}
```

Comment:
I think the UiOrderType is what should be used instead of a local flag https://github.com/cowprotocol/cowswap/blob/a2f8def2f203301885e4548b49f73336abbabe21/apps/cowswap-frontend/src/utils/orderUtils/getUiOrderType.ts#L14

Reply:
Good point. 

Leandro, one nit for this `UiOrderType` is that probably the Ui should be at the end (as sufix).
If we plan to reuse these UI types in explorer, we might want to move to a more central module.

Reply:
Great suggestion, thanks! Fixed